scripts/category_domains.py

```diff
@@ -0,0 +1,184 @@
+import json
+import os
+import sys
+import urllib2
+
+from trackingprotection_tools import DisconnectParser
+from lists2safebrowsing import (
+    canonicalize,
+    get_domains_from_category_filters,
+    get_domains_from_filters
+)
+
+DISCONNECT_MAPPING = os.path.join(
+    os.path.dirname(__file__), 'disconnect_mapping.json')
```

Comment:
@dave-light, this and anything referencing `DISCONNECT_MAPPING` should be removed from the code since with https://github.com/mozilla-services/shavar-list-creation/pull/121 `disconnect_mapping.json` will no longer be used.